heapster.go

```diff
@@ -34,16 +34,17 @@ import (
 )
 
 var (
-	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
-	argSinkFrequency   = flag.Duration("sink_frequency", 10*time.Second, "Frequency at which data will be pushed to sinks")
-	argCacheDuration   = flag.Duration("cache_duration", 4*time.Minute, "The total duration of the historical data that will be cached by heapster.")
-	argUseModel        = flag.Bool("use_model", true, "When true, the internal model representation will be used")
-	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
-	argPort            = flag.Int("port", 8082, "port to listen to")
-	argIp              = flag.String("listen_ip", "", "IP to listen on, defaults to all IPs")
-	argMaxProcs        = flag.Int("max_procs", 0, "max number of CPUs that can be used simultaneously. Less than 1 for default (number of cores)")
-	argSources         manager.Uris
-	argSinks           manager.Uris
+	argStatsResolution     = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
+	argSinkFrequency       = flag.Duration("sink_frequency", 10*time.Second, "Frequency at which data will be pushed to sinks")
+	argCacheDuration       = flag.Duration("cache_duration", 4*time.Minute, "The total duration of the historical data that will be cached by heapster.")
+	argRunningInKuberentes = flag.Bool("running_in_kubernetes", true, "When true, the kubernetes-specific api will be enabled")
```

Comment:
Can we somehow autodetect this? For example, if one of the sources is kubernetes, we can use that as an hint. Default values for flags like this affect usability. WDYT?